app/scripts/modules/netflix/feedback/feedback.modal.controller.ts

```diff
@@ -0,0 +1,99 @@
+import { module } from 'angular';
+import {IModalService, IModalServiceInstance} from 'angular-ui-bootstrap';
+
+import { AUTHENTICATION_SERVICE, AuthenticationService } from 'core/authentication/authentication.service';
+
+
+class FeedbackModalController implements ng.IComponentController {
+  static get $inject () {
+    return [
+      '$scope',
+      '$location',
+      '$http',
+      '$uibModalInstance',
+      'settings',
+      'authenticationService'
+    ];
+  }
+
+  constructor (private $scope: any,
```

Comment:
Also, it might be worth moving everything off `$scope` and on to the controller (i.e. `this`), then setting `controllerAs: 'ctrl'` when opening the modal. We've done that in most places as we've refactored older code - might as well do it here.